pysembench/handler.py

```diff
@@ -22,18 +24,41 @@ class Pykg2tblHandler(TaskHandler):
     ...
 
 
+class PyshaclHandler(TaskHandler):
+    def handle(self, task):
+        conforms, _, _ = validate(
+            data_graph=os.path.join(
+                task.input_data_location, task.config["data_graph"]
+            ),
+            shacl_graph=os.path.join(
+                task.sembench_data_location, task.config["shacl_graph"]
+            ),
+            data_graph_format="ttl",
+            shacl_graph_format="ttl",
+            inference="rdfs",
+            debug=True,
+        )
+        assert conforms, (
+            "pyshacl validation failed for "
+            f"data graph \"{task.config['data_graph']}\" "
+            "with "
+            f"shape graph \"{task.config['shacl_graph']}\""
+        )
+        return conforms
+
+
 class PysubytHandler(TaskHandler):
     def handle(self, task):
         """Construct a shell command based on the Task attributes and run it
         via a subprocess call.
         """
         input = task.config.get("input") or ""
         if input:
-            input = f"--input {os.path.join(task.input_data_location, input)}"
+            input = (
+                f'--input "{os.path.join(task.input_data_location, input)}"'
+            )
 
-        output = os.path.join(
-            task.output_data_location, (task.config.get("output") or "<uuid>")
-        )
+        output = os.path.join(task.output_data_location, task.config["output"])
```

Comment:
use pathlib's Path 
input_path = Path(task.input_data_location) / input

Reply:
I've tried to change it, but `pyshacl.validate` expects a string instead of a Path object, so I'm using `os.path` anyway. It seems a bit unnecessary to introduce pathlib just for this.